### PR TITLE
John Dave Decano - Technical Examination

### PR DESCRIPTION
**Pull request by johndavedecano**

### CODE REVIEW

**Comment by zakame** [comment]
Commit messages are all just `Changes`?

File: README.md [hunk 1]
```diff
@@ -1,23 +1,45 @@
-#+TITLE: API Org App
```
Comment (zakame) [inline README.md]: Conversion to Markdown not really needed here

**Comment by zakame** [comment]
@johndavedecano this app runs on what particular Node.js version?  Apparently I need ES6 support so likely v4.4.5?

**Comment by johndavedecano** [comment]
after npm install it will also install a babel transpiler that will able to support ES6

**Comment by johndavedecano** [comment]
i believe node 4+ will work i am using 4.4 as well

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "presets": ["es2015", "stage-0", "react"],
```
Comment (zakame) [inline .babelrc]: Seems to be lacking a corresponding dependency to `babel-preset-react` in `package.json`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "redux": "^3.5.2",
+    "superagent": "^2.0.0"
+  },
+  "devDependencies": {
+    "babel-core": "^6.9.1",
+    "babel-eslint": "^6.0.4",
+    "babel-plugin-react-transform": "^2.0.2",
+    "babel-plugin-transform-runtime": "^6.9.0",
+    "babel-preset-es2015": "^6.9.0",
+    "babel-preset-stage-0": "^6.5.0",
```
Comment (zakame) [inline package.json]: Perhaps needs `babel-preset-react` as dependency, see https://github.com/yoyo-holdings/api-org-app/pull/3/files#r66913882

File: README.md [hunk 4]
```diff
@@ -1,23 +1,45 @@
...
+
+##Installation
+  1. Clone repository git clone https://github.com/johndavedecano/api-org-app.git
+  2. Configure your mongoDB database URI and credentials at config.js.
+
+```
+      # Install npm packages
+      npm install
+      # Build Application
+      npm run build-production
```
Comment (zakame) [inline README.md]: Seems the `package.json` needs update:

```
θ87° [zakame:~/src … er-emacs/api-org-app] johndavedecano 2m22s ± npm run build-production                                                                 

> yoyojs@1.0.0 build-production /home/zakame/src/alter-emacs/api-org-app
> webpack --config webpack.config.production.js

Hash: 95a0e088ed43c4b9853b
Version: webpack 1.13.1
Time: 56ms
   [0] multi main 28 bytes {0} [built] [1 error]
    + 1 hidden modules

ERROR in The node API for `babel` has been moved to `babel-core`.
 @ multi main
```

Needed to run `npm install babel-loader --save-dev` prior.

File: .babelrc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  "env": {
+      "development": {
+          "plugins": [
+            "transform-runtime",
+            ["react-transform", {
+              "transforms": [{
+                "transform" : "react-transform-hmr",
```
Comment (zakame) [inline .babelrc]: Also probably another dependency here for `react-transform-hmr` in `package.json`.

File: package.json (hunk 3 above)
Comment (zakame) [inline package.json]: Also dep on `react-transform-hmr`

**Comment by zakame** [comment]
Yep, babel transpiler works, I even got it on node 0.12.7.  :+1:

**Comment by zakame** [comment]
`public/js/bundle.js` probably doesn't need to be committed.

**Comment by johndavedecano** [comment]
Yep, i just include so that it can still run without building it again

**Comment by johndavedecano** [comment]
Let me know if you were able to install and run it.

**Comment by zakame** [comment]
Yep, I was able to get it up and running, thanks :+1: 

However, I see this only implements TODOs, but not notes.

**Comment by johndavedecano** [comment]
Ok just mark it as failed then if necessary

**Comment by johndavedecano** [comment]
thanks, i apologize i am knowledgable at developing using notes

**Pull request closed without merging**